cmd/maya-agent/k8s/types.go

```diff
@@ -13,13 +13,16 @@ import (
 
 // #############################################################################
 
+
```

Comment:
Just a minor pick,we can probably remove this extra line.

Reply:
👍 Would do that. Thanks!